src_python/fl_client.py

```diff
@@ -205,32 +224,39 @@ def run(self):
     if 'epochs' not in args.keys():
         args['epoch'] = 10
 
-    logging.warning('####################################### New Training Session #######################################')
-    logging.info('Client started, graph ID %s, partition ID %s, epochs %s',args['graph_id'],args['partition_id'],args['epochs'])
+    logging.warning(
+        '################################# New Training Session #################################')
+    logging.info('Client started, graph ID %s, partition ID %s, epochs %s',
+                 args['graph_id'], args['partition_id'], args['epochs'])
 
-    path_nodes = args['path_nodes'] + args['graph_id'] + '_nodes_' + args['partition_id'] + ".csv"
-    nodes = pd.read_csv(path_nodes,index_col=0)
+    path_nodes = args['path_nodes'] + args['graph_id'] + \
+        '_nodes_' + args['partition_id'] + ".csv"
```

Comment:
We should stick to use same String delimiter format. The standard is to use single quotes. https://docs.ckan.org/en/ckan-2.1.5/python-coding-standards.html If we are using a single quote within the string we should use double quotes only for such strings. In the case of `".csv"` we should change it to use single quotes.